frame/bonded-finance/src/lib.rs

```diff
@@ -312,10 +312,12 @@ pub mod pallet {
 			offer: BondOfferOf<T>,
 			keep_alive: bool,
 		) -> Result<T::BondOfferId, DispatchError> {
-			let offer_id = BondOfferCount::<T>::try_mutate(|offer_id| -> Result<T::BondOfferId, DispatchError> {
-				*offer_id = offer_id.safe_add(&T::BondOfferId::one())?;
-				Ok(*offer_id)
-			})?;
+			let offer_id = BondOfferCount::<T>::try_mutate(
+				|offer_id| -> Result<T::BondOfferId, DispatchError> {
+					*offer_id = offer_id.safe_add(&T::BondOfferId::one())?;
+					Ok(*offer_id)
+				},
```

Comment:
Perhaps somewhat unrelated to this PR, but we use this "try increment storage" pattern a lot, maybe we could extract it to a helper function in `composable-helpers`?

Reply:
I think it's pretty fair to extract that, yes